### PR TITLE
Create es.js

### PR DESCRIPTION
**Pull request by RAVMN**
Translation made by Richard von Moltke (ravmn) with the help from Virgilio Ruilova, reviewed by Joaquín Haeussler (josotrix), the three from the Mozilla Chile community.

### CODE REVIEW

File: locales/es.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+	"Websites filled: ": "Sitios web rellenados: ",
+	"Days since last joined: ": "Días desde la última conexión: ",
+	"Lessons completed by user": "Lecciones completadas por usuario",
+	" completed": " completadas",
+	"Lessons completed by lesson": "Lecciones completas por lección",
+	"Perform the following tasks in JavaScript:": "Realizar las siguientes tareas en JavaScript:",
+	"Not yet completed": "Aún no completado",
+	"So far so good.": "Hasta ahora, todo bien.",
+	"Do not have any syntax errors.": "No cometas errores de sintaxis.",
+<<<<<<< HEAD
```
Comment (bbondy) [inline locales/es.js]: Looks like there's a merge conflict here, I think you want both included. Could you fix?

**Comment by bbondy** [comment]
This is really great, thanks so much for all of the work!
See comment above, and after that I'll merge it in.

File: locales/es.js (hunk 1 above)
Comment (RAVMN) [inline locales/es.js]: Sorry, I don't know what is needed :/

Comment (bbondy) [inline locales/es.js]: Basically just these lines need to be deleted:
<<<<<<< HEAD on line 207
======= on line 212
and 
>>>>>>> bf08ee362e6380fada6aa49e7de33b5d44e99012
on line 218

After you remove them you can git add -u 
Then commit --amend

**Comment by pquentin** [comment]
@bbondy Could you have forgotten to merge this PR?

**Comment by bbondy** [comment]
Thanks! I didn't get re-pinged when you updated. Merging.

**Pull request merged**